lib/MusicBrainz/Server/Data/Event.pm

```diff
@@ -226,6 +226,7 @@ sub find_by_artist
 
 sub _order_by {
     my ($self, $order) = @_;
+    $order = ($order eq "") ? "-date" : $order;
```

Comment:
This solution is good, but just a minor nitpick: `$order` will be `undef` unless the URL explicitly contains `?order=`, and  `undef eq ''` will cause perl to generate a warning in our logs every time someone requests the page. You can provide a fallback for `undef` using the `//` operator:
```suggestion
    $order = (($order // "") eq "") ? "-date" : $order;
```

Reply:
That's cool, I never known you could do that with Perl